edx_event_bus_kafka/internal/tests/test_consumer.py

```diff
@@ -82,7 +83,45 @@ def setUp(self):
             error=None,
         )
         self.mock_signal = Mock(event_type=self.signal_type, init_data={})
-        self.event_consumer = KafkaEventConsumer('test_topic', 'test_group_id', self.mock_signal)
+        self.event_consumer = KafkaEventConsumer('some-topic', 'test_group_id', self.mock_signal)
+
+    @override_settings(
+        EVENT_BUS_KAFKA_SCHEMA_REGISTRY_URL='http://localhost:12345',
+        EVENT_BUS_KAFKA_BOOTSTRAP_SERVERS='localhost:54321',
+        EVENT_BUS_TOPIC_PREFIX='prod',
```

Comment:
[question] What is the benefit to including prefix logic in these tests?

Reply:
Proving that the consumer and producer actually use the new method, basically. :-)